examples/helloWorld.js

```diff
@@ -11,7 +11,7 @@ apiModel.registerPublicRoute(
     , 'Hello World Example Route'
     , '/'
     , function(req, res, next) { res.send('Hello World'); next();}
-    , {}
+    , [{}]
```

Comment:
Probably should add something here to actually show how it will generate swagger